helpers/deploymentConfig.ts

```diff
@@ -0,0 +1,121 @@
+export type PreconfiguredAddresses = { [contract: string]: string };
+
+interface BridgeConfig {
+  [networkName: string]: {
+    methods: { method: string; args: any[] }[];
+  };
+}
+
+const SEPOLIA_MULTISIG = "0x94fa6078b6b8a26f0b6edffbe6501b22a10470fb";
+
+export const preconfiguredAddresses = {
+  hardhat: {
+    VTreasury: "account:deployer",
+  },
+  bsctestnet: {
```

Comment:
Rather than hardcode these we should get them from the packages

Reply:
[9384b3c](https://github.com/VenusProtocol/token-bridge/pull/2/commits/9384b3cb5a872fb1edad6f741112d51c4c699f2e)

Reply:
#1